boards/firefly/Makefile.include

```diff
@@ -0,0 +1,11 @@
+USEMODULE += remote-common
+
+# define the default port depending on the host OS
+PORT_LINUX  ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+include $(RIOTBOARD)/remote-common/Makefile.include
+
+# Board provides 2 network interfaces and should be 
+#GNRC_NETIF_NUMOF := 2
```

Comment:
I think it would make sense to move this into the board's `Makefile.dep` and set the value according to the network devices that are actually enabled.

Reply:
Just a quick question about that. Is there a possibilty to choose which network device to enable at runtime? For example, I have finished the CC1200 network driver for the Zoul Module and the Firefly is able to use both interfaces at the same time, so booting up both and using both interfaces is possible when leaving `GNRC_NETIF_NUMOF := 2` in. But when i say `GNRC_NETIF_NUMOF := 1`, it always enables the CC1200 network device, if I leave the `USEMODULE += CC1200` in. So what I mean is, if it is possible to have both network devices, meaning the `CC1200` and the `CC2538 RF` enabled, while just using one interface by using `GNRC_NETIF_NUMOF := 1` and switch the corresponding device at runtime?

Reply:
But maybe its best to leave it out for now and add it later when the CC1200 driver has been merged.